numpy/_core/src/npysort/highway_qsort.hpp

```diff
@@ -11,6 +11,13 @@ namespace np { namespace highway { namespace qsort_simd {
 NPY_CPU_DISPATCH_DECLARE(template <typename T> void QSort, (T *arr, npy_intp size))
 NPY_CPU_DISPATCH_DECLARE(template <typename T> void QSelect, (T* arr, npy_intp num, npy_intp kth))
 
+
+#ifndef NPY_DISABLE_OPTIMIZATION
+    #include "highway_qsort_16bit.dispatch.h"
+#endif
+NPY_CPU_DISPATCH_DECLARE(template <typename T> void QSort, (T *arr, npy_intp size))
+NPY_CPU_DISPATCH_DECLARE(template <typename T> void QSelect, (T* arr, npy_intp num, npy_intp kth))
```

Comment:
`QSelect` isn't implemented yet, right? I suppose it can come after https://github.com/google/highway/pull/1710.